File: app/src/main/java/com/sergiigalai/chargetimer/bugreport/BugReportActivity.java
package com.sergiigalai.chargetimer.bugreport;

import android.os.Bundle;
import android.text.method.ScrollingMovementMethod;
import android.widget.TextView;

import com.sergiigalai.chargetimer.R;

public class BugReportActivity extends BaseActivity
{
    TextView error;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_bugreport);

        error = findViewById(R.id.error);
        error.setMovementMethod(new ScrollingMovementMethod());
        error.setText(getIntent().getStringExtra("error"));
    }
}

File: EvChargeTimer/app/src/main/java/com/chebuso/chargetimer/bugreport/BugReportActivity.kt
package com.chebuso.chargetimer.bugreport

import android.os.Bundle
import android.text.method.ScrollingMovementMethod
import android.widget.TextView
import com.chebuso.chargetimer.R
import com.chebuso.chargetimer.controls.BaseActivity


class BugReportActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_bugreport)
        findViewById<TextView>(R.id.error).apply {
            movementMethod = ScrollingMovementMethod()
            text = intent.getStringExtra("error")
        }
    }
}

